src/NuProj.Targets/NuProj.targets

```diff
@@ -148,7 +150,7 @@
     <ProjectReference>
       <PackageDirectory>Lib</PackageDirectory>
       <TargetFramework></TargetFramework>
-      <Targets>$(ProjectOutputGroups)</Targets>
+      <ProjectOutputGroups>$(ProjectOutputGroups)</ProjectOutputGroups>
```

Comment:
The problem I see is that **none** of targets listed in `ProjectOutputGroups` (or `Targets` previously) depend on `Build` target. If I kept using `Targets`, none of the project references would get built when using command line, meaning that none of the dependencies would get produced, which would cause build to fail.

On the other side I think that keeping `ProjectOutputGroups` gives users important extensibility point, when they need to package result from target of some obscure project type.